packages/Sortable/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+- Added isSemantic false to Sortable.Item delete button
```

Comment:
Should add this to the changeset instead of directly in the CHANGELOG.  (`yarn changeset`).